### PR TITLE
[3D] pegman mode

### PR DESCRIPTION
**Pull request by fredj**
demo: https://mf-geoadmin3.dev.bgdi.ch/pegman/?dev3d=true

### CODE REVIEW

**Comment by fredj** [comment]
todo: update icons

**Comment by fredj** [comment]
Thanks for any review

**Comment by gjn** [comment]
Amazing. This is going to rule. I like the icons!

First thing: next to pegman, we need to have visbile help that shows keyboard bindings and how to use the pegman mode[s].

The mouse isn't used anymore, which I think takes aways from the exprience. It's not easy to 'look around' using keyboards...but would like to know input from others as well.

Keys to adapt 'height above ground' and 'speed' would be great as well (and show actual values next to icons).

**Comment by gjn** [comment]
Code LGTM.

**Comment by cedricmoullet** [comment]
[] Clicking on "f" activates the fly mode. Superpegman should also be visible after clicking on f
[] With shift, pegman moves/flies faster. Could it be more fast ? Or could we use ctrl-shift in order to have a "FA-18" speed ?
[] After clicking on pegman, please add a text box containing information on how to fly (keys etc...)
[] Rename SuperPegman.png to super_pegman.png
[] Mobile css

**Comment by fredj** [comment]
updated todo list:
 - [ ] use mouse to look around in fps mode
 - [ ] left/right keys moves pegman instead of moving the camera

File: src/components/controls3d/fps.js [hunk 1]
```diff
@@ -0,0 +1,431 @@
+goog.provide('fps');
+
+
+
+/**
+ * @param {!Cesium.Scene} scene
+ * @param {!Angular.Scope} scope
+ * @constructor
+ */
+function FPS(scene, scope) {
```
Comment (oterral) [inline src/components/controls3d/fps.js]: Shouldn't it be a service ? I'm pretty sure we will have the need to know which state is the FPS in other directives in the future.

**Comment by fredj** [comment]
@gjn I've added the help div and updated the demo link: https://mf-geoadmin3.dev.bgdi.ch/pegman/?dev3d=true

See also #2762 to track down the improvements of the pegman mode

**Comment by fredj** [comment]
Before merging, I need to fix the altitude in pegman mode.
See https://github.com/geoadmin/mf-geoadmin3/blob/master/src/js/GaCesium.js#L53

**Comment by gjn** [comment]
Let ne know when you're done.

**Comment by gjn** [comment]
oops. Ok, I'll review and merge.

**Comment by gjn** [comment]
Very fine work. Thanks alot.

**Pull request merged**